lib/library.rb

```diff
@@ -0,0 +1,45 @@
+class Library
+  attr_reader :authors, :books, :name, :checked_out_books
+
+  def initialize(name)
+    @name = name
+    @authors = []
+    @books = []
```

Comment:
I'd rather this be calculated dynamically since we have the info in @authors. Single source of truth.